skimage/data/tests/test_data.py

```diff
@@ -116,24 +107,8 @@ def test_cell():
     data.cell()
 
 
-@skipif(OFFLINE, reason='No internet connection')
 def test_cells_3d():
     """Needs internet connection."""
     path = fetch('data/cells.tif')
     image = io.imread(path)
     assert image.shape == (60, 256, 256)
-
-
-def test_data_dir():
-    astro = data.astronaut()
-    datadir = data.data_dir
-    assert 'astronaut.png' in os.listdir(datadir)
-
-
-@skipif(OFFLINE, reason='No internet connection')
-def test_download_all():
```

Comment:
Is there a good reason to remove this test?

Reply:
This test really doen't play with parallel testing.... affecting the global filesystem state is not fun.

Reply:
it also may induce a full download of all the files for the end users. not fun on our laptops when travelling with 4G connections.